moveit_ros/visualization/motion_planning_rviz_plugin/include/moveit/motion_planning_rviz_plugin/motion_planning_display.hpp

```diff
@@ -65,7 +67,7 @@ namespace Ogre
 class SceneNode;
 }
 
-namespace rviz
+namespace rviz_common
```

Comment:
Why are we changing the namespace?

Reply:
Rviz2 plugins use that namespace because most rviz core funcionalities and classes are now in rviz_common.